pkg/arvo/sys/lull.hoon

```diff
@@ -1813,17 +1814,130 @@
   +$  verb  ?(%odd)
   ::
   ::  +agent: app core
+  +$  agent-wefts  ?([%zuse %416])
   ::
-  ++  agent
+  ++  agent-any  $%  [[%zuse %416] agent-416]
+                 ==
+  ++  agent  agent-416
+  ::
+  ++  agent-416
     =<  form
     |%
     +$  step  (quip card form)
     +$  card  (wind note gift)
     +$  note
       $%  [%agent [=ship name=term] =task]
-          [%arvo note-arvo]
+          [%arvo userspace-note]
           [%pyre =tang]
       ==
+    +$  userspace-gift
+      $%  $>(%wake gift:behn)
+          $>(%heck gift:behn)
+          ::
+          $>(%wris gift:clay)
+          $>(%writ gift:clay)
+          $>(%tire gift:clay)
+          $>(%hill gift:clay)
+          $>(%mere gift:clay) :: kiln
+          $>(%cruz gift:clay)
+          $>(%croz gift:clay)
+          ::
+          $>(%blit gift:dill)
+          ::
+          $>(%bound gift:eyre)
+          ::
+          $>(%unto gift:gall)
+          ::
+          $>(%http-response gift:iris)
+          ::
+          $>(%private-keys gift:jael)
+          $>(%public-keys gift:jael)
+          $>(%turf gift:jael)
+          ::
+          $>(%arow gift:khan)
+      ==
+    +$  userspace-note
+      $%  $:  %a
+        $%  $>(%sift task:ames)
+            $>(%spew task:ames) :: set verbosity (we want this?)
+            $>(%cork task:ames) :: gall does this
+            $>(%stir task:ames) :: ?? helm does this
+            $>(%prod task:ames) :: ?? helm does this
+        ==  ==
+          $:  %b
+        $%  $>(%rest task:behn)
+            $>(%wait task:behn)
+            $>(%huck task:behn) :: gall does htis
```

Comment:
just because gall can %huck doesn't mean userspace should be able to -- since it's manipulating the duct call stack, I don't think userspace should be allowed to do it

Reply:
Gall passes a `%huck` from `++ap-kill-down`, eventually ending up in `+ap-move`, where the moves end up in `agent-moves` inside the `+ap` core which is now of type `userspace-move`. I would like the `%huck` moves to end up straight in `moves` under the `++mo` core, unsure of how to achieve this though.

https://github.com/urbit/urbit/blob/31e7a3acba69577ec96cd3eb15ca5f89afa41c5c/pkg/arvo/sys/vane/gall.hoon#L1508-L1515

https://github.com/urbit/urbit/blob/31e7a3acba69577ec96cd3eb15ca5f89afa41c5c/pkg/arvo/sys/vane/gall.hoon#L1201-L1203 

https://github.com/urbit/urbit/blob/31e7a3acba69577ec96cd3eb15ca5f89afa41c5c/pkg/arvo/sys/vane/gall.hoon#L886

https://github.com/urbit/urbit/blob/31e7a3acba69577ec96cd3eb15ca5f89afa41c5c/pkg/arvo/sys/vane/gall.hoon#L194

Reply:
As discussed out of band, I made agent-moves an intermediate type `(list gall-move)`, which includes some gifts and notes that are used by the Gall vane but are disallowed for userspace. 
https://github.com/urbit/urbit/blob/f9b1af32f47782bea67394490d612bc350732e43/pkg/arvo/sys/vane/gall.hoon#L48-L68